dietpi/dietpi-software

```diff
@@ -10464,30 +10464,31 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$INSTALLING_INDEX]} == 1 )); then
 
 			# - run syncthing to create cert/config and exit
-			syncthing -generate="/$HOME/.config/syncthing"
+			syncthing -generate="$FP_DIETPI_USERDATA_DIRECTORY"/syncthing
 
 			# - Disable automatic upgrades
-			sed -i '/<\/autoUpgradeIntervalH>/c\        <autoUpgradeIntervalH>0<\/autoUpgradeIntervalH>' "$HOME"/.config/syncthing/config.xml
+			sed -i '/<\/autoUpgradeIntervalH>/c\        <autoUpgradeIntervalH>0<\/autoUpgradeIntervalH>' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# - Allow external access (LAN).
-			sed -i '/:8384<\/address>/c\        <address>0.0.0.0:8384<\/address>' "$HOME"/.config/syncthing/config.xml
+			sed -i '/:8384<\/address>/c\        <address>0.0.0.0:8384<\/address>' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# - Set default folder
-			sed -i '/label=\"Default Folder/c\    <folder id=\"0000-0000\" label=\"DietPi user data\" path=\"/mnt/dietpi_userdata/\" type=\"readwrite\" rescanIntervalS=\"60\" ignorePerms=\"false\" autoNormalize=\"true\">' "$HOME"/.config/syncthing/config.xml
+			sed -i '/label=\"Default Folder/c\    <folder id=\"0000-0000\" label=\"syncthing data\" path=\"/mnt/dietpi_userdata/syncthing_data\" type=\"readwrite\" rescanIntervalS=\"60\" ignorePerms=\"false\" autoNormalize=\"true\">' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# - Disable browser starting
-			sed -i '/<\/startBrowser>/c\        <startBrowser>false<\/startBrowser>' "$HOME"/.config/syncthing/config.xml
+			sed -i '/<\/startBrowser>/c\        <startBrowser>false<\/startBrowser>' "$FP_DIETPI_USERDATA_DIRECTORY"/syncthing/config.xml
 
 			# services
 			cat << _EOF_ > /etc/systemd/system/syncthing.service
 [Unit]
 Description=Syncthing
 After=network.target
 
+
 [Service]
 Type=simple
-ExecStart=/usr/bin/syncthing -logfile=/var/log/syncthing.log -logflags=3
```

Comment:
I'd prefer to keep the log files in ```/var/log```, to gain the benefit of RAMlog (installed by default) performance.

Reply:
@Fourdee 

> Need to generate the log files during install and set permissions.

Want me to do it?

I was wondering if there was an easy way to make diepi the default user to run some apps.
The vulnerability you get by having an untrusted application open to the internet as root it too high.

Reply:
@joaofl 

> Want me to do it?

Would be grateful if you could 👍 

> I was wondering if there was an easy way to make diepi the default user to run some apps.
The vulnerability you get by having an untrusted application open to the internet as root it too high.

Planned: https://github.com/Fourdee/DietPi/issues/896

Reply:
@Fourdee 
What if the dietpi user had r/w access to the folder `/var/log/` ? Is that possible/viable?
Then it would be it much easier to get other apps running as dietpi as well.